code/client/cl_console.c

```diff
@@ -566,7 +566,7 @@ void CL_ConsolePrint( const char *txt ) {
 		}
 
 		// word wrap
-		if ( l != con.linewidth && ( con.x + l >= con.linewidth ) ) {
+		if ( l != con.linewidth && ( con.x + l >= con.linewidth ) && (l + (con_timestamp->integer ? 10 : 0) <= con.linewidth)) {
```

Comment:
Maybe would be better to change the // count word length section slightly above? On the other hand, don't need to account for con_timestamp being initialized if word wrap is simply disabled for very long words.